app/views/exercise_collections/statistics.html.slim

```diff
@@ -15,3 +15,27 @@ h1 = @exercise_collection
       .legend-entry
         div(class="box #{klass}")
         .box-label = label
+
+h4.mt-4 = t('activerecord.attributes.exercise_collections.exercises')
+.table-responsive#exercise-list
+  table.table
```

Comment:
What about using a sortable table? Compare the following code snippet: https://github.com/openHPI/codeocean/blob/fd4a1f4fa5a3075fc8444d0a1790430e1c9f25d6/app/views/exercises/statistics.html.slim#L41-L42

```suggestion
  table.table.sortable
```